[Bug][docker-compose] devlake container keeps restarting

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.


### What happened

I used the instructions to install with docker, I used podman-compose instead of docker-compose.
I am unable to access the config ui and when examining the containers, I noticed that devlake_devlake_1 keeps restarting. Upon trying to attach, I saw this message:

`Version:  v0.17.0-beta4@
time="2023-06-13 23:32:39" level=warning msg="Failed to create filestream for logs. Logs will not be piped to files.
        caused by: open /app/logs/devlake.log: permission denied
        Wraps: (2) open /app/logs/devlake.log
        Wraps: (3) permission denied
        Error types: (1) *hintdetail.withDetail (2) *fs.PathError (3) syscall.Errno"
Attempting to write configuration to .env file.
panic: open ./.env: permission denied
Wraps: (2) open ./.env
Wraps: (3) permission denied
Error types: (1) *hintdetail.withDetail (2) *fs.PathError (3) syscall.Errno

goroutine 1 [running]:
main.main()
        /app/server/main.go:36 +0x93
`

In addition, browsing to http://localhost:4000/ I see the following message in the devlake_config-ui_1 console:
`
10.89.0.1 - - [13/Jun/2023:23:36:25 +0000] "GET /assets/index-14ed113a.js HTTP/1.1" 200 1424102 "http://localhost:4000/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36" "-"
2023/06/13 23:36:25 [error] 8#8: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.89.0.1, server: localhost, request: "GET /api/version HTTP/1.1", upstream: "http://10.89.0.8:8080/version", host: "localhost:4000", referrer: "http://localhost:4000/"

`

### What do you expect to happen

I expect to see the config ui and possibly the devlake container remain running and stable

### How to reproduce

`podman-compose up -d`
Use: https://devlake.apache.org/docs/GettingStarted/DockerComposeSetup
with: https://github.com/apache/incubator-devlake/releases/tag/v0.17.0-beta4 

### Anything else

Also tried docker-compose with sudo
also tried podman-compose with sudo
I'm running on Fedora 37

Thanks :heart: 

### Version

v0.17.0-beta4 

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/devops/releases/lake-v0.17.0/docker-compose.yml
+++ b/devops/releases/lake-v0.17.0/docker-compose.yml
@@ -54,7 +54,7 @@ services:
       - 8080:8080
     restart: always
     volumes:
-      - ./.env:/app/.env
+      - ./.env:/app/.env:z
       - ./logs:/app/logs
     environment:
       LOGGING_DIR: /app/logs

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -75,7 +75,7 @@ services:
       - "127.0.0.1:8080:8080"
     restart: always
     volumes:
-      - ./.env:/app/.env
+      - ./.env:/app/.env:z
       - ./logs:/app/logs
     environment:
       LOGGING_DIR: /app/logs

